service/k8sclient/k8sclient.go

```diff
@@ -584,16 +597,20 @@ func (c *K8sClient) UpdateXtraDBCluster(ctx context.Context, params *XtraDBParam
 
 // DeleteXtraDBCluster deletes Percona XtraDB cluster with provided name.
 func (c *K8sClient) DeleteXtraDBCluster(ctx context.Context, name string) error {
+	operators, err := c.CheckOperators(ctx)
+	if err != nil {
+		return err
+	}
 	res := &pxc.PerconaXtraDBCluster{
 		TypeMeta: common.TypeMeta{
-			APIVersion: pxcAPIVersion,
+			APIVersion: c.getAPIVersionForPXCOperator(operators),
```

Comment:
THIS HAS TO BE VERSION OF THE OPERATOR WE DEPLOYED THE CLUSTER WITH, RIGHT? 
TODO: CHECK THIS.